.htaccess

```diff
@@ -0,0 +1,31 @@
+# Save this file as UTF-8, or at least, plain ASCII text.
+# These are considered current best practice.
+# Don't change them unless you're having problems on your site.
+php_flag auto_detect_line_endings on
+php_value default_charset UTF-8
+
+# Change all below to reflect your needs
+
+# 'register_globals' was removed in PHP 5.4
+# Remove this line if you're using PHP >= 5.4.
+php_flag register_globals off
+
+# 'magic_quotes' were removed in PHP 7.
+# Remove these lines if you're using PHP >= 7.
+php_flag magic_quotes_gpc Off
```

Comment:
This would be nice... to be able to force this setting off right from within webcalendar.